airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -153,6 +153,12 @@
   dockerImageTag: 0.2.3
   documentationUrl: https://hub.docker.com/r/airbyte/source-twilio-singer
   icon: twilio.svg
+- sourceDefinitionId: b9dc6155-672e-42ea-b10d-9f1f1fb95ab1
+  name: Twilio
+  dockerRepository: airbyte/source-twilio-singer
+  dockerImageTag: 0.1.0
+  documentationUrl: https://hub.docker.com/r/airbyte/source-twilio
+  icon: twilio.svg
```

Comment:
can you remove the icon? will break users on previous versions of airbyte

also this should replace the previous twilio singer version